src/wp-includes/block-supports/layout.php

```diff
@@ -630,7 +630,16 @@ function wp_render_layout_support_flag( $block_content, $block ) {
 
 	$class_names        = array();
 	$layout_definitions = wp_get_layout_definitions();
-	$container_class    = wp_unique_id( 'wp-container-' );
+
+	/*
+	 * We use an incremental ID that is independent per prefix to make sure that
+	 * rendering different numbers of blocks doesn't affect the IDs of other
+	 * blocks. We need this to make the CSS class names stable across paginations
+	 * for features like the enhanced pagination of the Query block.
+	*/
```

Comment:
```suggestion
	 */
```
Minor alignment issue.